src/components/BookStatusPanel/BookStatusPanel.tsx

```diff
@@ -4,12 +4,28 @@ import { Status } from './BookStatus/Status'
 
 import styles from './BookStatusPanel.module.css'
 
-export const BookStatusPanel = () => {
+type BookStatusPanelProps = {
+  isAdmin?: boolean
+}
+
+export const BookStatusPanel: React.FC<BookStatusPanelProps> = ({
+  isAdmin,
+}) => {
   return (
-    <div className={styles.statusPanel}>
-      <BookStatus status={Status.APPROVED} />
-      <BookRating rating={4.5} />
-    </div>
+    <>
+      {!isAdmin && (
```

Comment:
Because the difference is only in styles, you can handle the className switching like this to reduce boilerplate


```
 <div className={isAdmin ? styles.statusPanelAdmin: styles.statusPanel}>
      <BookStatus status={Status.RENTED} />
      <BookRating rating={4.5} />
 </div>
```